packages/playwright-core/src/cli/program.ts

```diff
@@ -571,7 +571,6 @@ async function codegen(options: Options & { target: string, output?: string, tes
     mode: 'recording',
     testIdAttributeName,
     outputFile: outputFile ? path.resolve(outputFile) : undefined,
-    handleSIGINT: false,
```

Comment:
is it related to this PR?

Reply:
Yes, I don't think we need the handleSIGINT: true mode for recorder, at all, so removing.